tests/gha-run-k8s-common.sh

```diff
@@ -264,3 +264,54 @@ function deploy_k8s() {
 
 	echo "::endgroup::"
 }
+
+function install_snapshotter(){
+
+    if [ "${SNAPSHOTTER}" == "nydus" ]; then
+        echo "Skipping install for ${SNAPSHOTTER} snapshotter"
+        return 0
+    fi
+
+	case "${SNAPSHOTTER}" in
+		nydus)
+			nydus_snapshotter_version="$(get_from_kata_deps "externals.nydus-snapshotter.version")"
+			install_nydus_snapshotter "$nydus_snapshotter_version" "source";;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+} 
+
+function cleanup_nydus_snapshotter(){
```

Comment:
nit: Add space before function block for consistency
```suggestion
function install_snapshotter() {
```